Kconfig

```diff
@@ -1,11 +1,6 @@
 # Kconfig file for LVGL v7.8.1
 
 menu "LVGL configuration"
-    
-    config LV_ATTRIBUTE_FAST_MEM_USE_IRAM
-        bool "Set IRAM as LV_ATTRIBUTE_FAST_MEM"
-        help
-            Set this option to configure IRAM as LV_ATTRIBUTE_FAST_MEM
```

Comment:
This exists on esp32 processors and should not be removed.

Reply:
That's correct; it [instructs the ESP32 build system](https://github.com/lvgl/lvgl/blob/7d3a6c8de4303f69582a3939d36405a06b376ed0/CMakeLists.txt#L31-L33) to set `LV_ATTRIBUTE_FAST_MEM`, which [does exist](https://github.com/lvgl/lvgl/blob/7d3a6c8de4303f69582a3939d36405a06b376ed0/lv_conf_template.h#L281).

Reply:
Ok, I will remove this patch from PR.

Reply:
Maybe we can add a comment about that on the Kconfig file (the option being used for ESP32 processors)